language_service/src/compilation.rs

```diff
@@ -46,6 +47,8 @@ pub(crate) enum CompilationKind {
     /// one or more sources, and a target profile.
     OpenProject {
         package_graph_sources: PackageGraphSources,
+        /// a human-readable name for the package (not a unique URI -- meant to be read by humans)
+        human_readable_name: Arc<str>,
```

Comment:
Just cause we refer to this as the "friendly name" in `project.rs`, how about `friendly_name`? Since you also used `pretty_name` below, I guess this went through a few renames... I think they're all good options but maybe we should stick to one term.